packages/story-editor/src/karma/fixture/containers/documentPanel/categoriesAndTags.js

```diff
@@ -35,7 +35,7 @@ export class CategoriesAndTags extends Container {
   }
 
   get addNewCategoryButton() {
-    return this.getByRole('button', { name: /Add New Category/ });
+    return this.queryByRole('button', { name: /Add New Category/ });
```

Comment:
what's the difference between these two?

Reply:
queryBy let's the button not be found and doesn't error. I had added this in for when I thought that capabilities were a check coming from `config` rather than the stories response. i don't think it's necessary now.